client/components/PubReleaseDialog/PubReleaseDialog.tsx

```diff
@@ -236,10 +300,14 @@ const PubReleaseDialog = (props: Props) => {
 				{renderReleaseResult()}
 			</div>
 			<div className={Classes.DIALOG_FOOTER}>
-				<div className={Classes.DIALOG_FOOTER_ACTIONS}>
-					{createdRelease && renderPostReleaseButtons()}
-					{!createdRelease && renderPreReleaseButtons()}
-				</div>
+				{hasSuggestions() ? (
```

Comment:
this line should also check that the suggested edits feature flag is enabled for this community